src/Fixer/SpecifyArgSeparatorFixer.php

```diff
@@ -0,0 +1,149 @@
+    }
+
+    public function isRisky(): bool
+    {
+        return true;
+    }
+
+    public function fix(\SplFileInfo $file, Tokens $tokens): void
+    {
+        foreach ($tokens as $index => $token) {
+            if ($token->isGivenKind(T_STRING) && $token->getContent() === 'http_build_query') {
+                $previousIndex = $tokens->getPrevMeaningfulToken($index);
+
+                // Function imports and function definitions should not be affected
+                if ($previousIndex !== null
+                    && ($tokens[$previousIndex]->isGivenKind(CT::T_FUNCTION_IMPORT)
+                        || $tokens[$previousIndex]->isGivenKind(T_FUNCTION))) {
+                    return;
+                }
+
+                $this->fixFunction($tokens, $index);
+            }
+        }
+    }
+
+    public function getName(): string
+    {
+        return mb_strtolower(self::class);
+    }
+
+    public function getPriority(): int
+    {
+        return 0;
+    }
+
+    public function supports(\SplFileInfo $file): bool
+    {
+        return true;
+    }
+
+    private function fixFunction(Tokens $tokens, int $functionIndex): void
+    {
+        $startBraceIndex = $tokens->getNextTokenOfKind($functionIndex, ['(']);
+        if ($startBraceIndex === null) {
+            return;
+        }
+
+        $endBraceIndex = $tokens->findBlockEnd(Tokens::BLOCK_TYPE_PARENTHESIS_BRACE, $startBraceIndex);
+        $ampToken = new Token([T_STRING, "'&'"]);
+
+        $commaIndexes = [];
+
+        for ($index = $startBraceIndex + 1; $index < $endBraceIndex; ++$index) {
+            $token = $tokens[$index];
+
+            // Parameter surrounded by brace, skip to its end
+            if ($token->equals('(')) {
+                $index = $tokens->findBlockEnd(Tokens::BLOCK_TYPE_PARENTHESIS_BRACE, $index);
+
+                continue;
+            }
+
+            // Array parameter, skip to its end
+            if ($token->isGivenKind(CT::T_ARRAY_SQUARE_BRACE_OPEN)) {
+                $index = $tokens->findBlockEnd(Tokens::BLOCK_TYPE_ARRAY_SQUARE_BRACE, $index);
+
+                continue;
+            }
+
+            if ($token->equals(',')) {
+                $commaIndexes[] = $index;
+
+                continue;
+            }
+        }
+
+        $paramsCount = count($commaIndexes) + 1;
+        $beforeEndBraceIndex = $tokens->getPrevNonWhitespace($endBraceIndex);
+
+        // When third parameter is already present, make sure it is not null or override its value
+        if ($paramsCount >= 3) {
+            $thirdParamTokenIndex = $tokens->getNextMeaningfulToken($commaIndexes[1]);
+            if ($thirdParamTokenIndex === null) {
+                return;
+            }
+            $thirdParamToken = $tokens[$thirdParamTokenIndex];
+
+            if ($thirdParamToken->isGivenKind(T_STRING) && $thirdParamToken->getContent() === 'null') {
+                $tokens->offsetSet($thirdParamTokenIndex, $ampToken);
```

Comment:
This section very difficult to read. What does it do?

I'd expect it to be bellow, with other "modifying" code